promscale/tobs/tobs.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Install the observability stack (tobs) for Kubernetes
+excerpt: Install tobs
+product: promscale
+keywords: [analytics, Kubernetes]
+tags: [install, k8s]
+related_pages:
+  - /promscale/:currentVersion:/recommendations/resource-recomm
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+# Install the observability stack (tobs) for Kubernetes
+The observability stack (tobs), is a pre-packaged distribution of observability
+tools and dashboard interfaces which can be installed on any existing Kubernetes
+cluster. It includes many of the most popular open-source observability tools
+including Prometheus, Grafana, Jaeager, TimescaleDB, and [other components][stack-components].
```

Comment:
```suggestion
including Prometheus, Grafana, TimescaleDB, and [other components][stack-components].
```

Jaeger is no longer part of the tobs stack so we can remove this.

Reply:
Ugh, didn't even notice that typo until now either 🤦‍♀️